### PR TITLE
Add CI setup

### PR DESCRIPTION
**Pull request by pierDipi**
As per title, build chart on presubmit and push on postsubmit

### CODE REVIEW

**Comment by pierDipi** [comment]
@ReToCode I guess this will not run until it's merge once?

**Review by ReToCode — CHANGES_REQUESTED** [review]

**Comment by ReToCode** [comment]
> @ReToCode I guess this will not run until it's merge once?

Yes I think it needs to be on the target branch.

File: .github/workflows/chart-build.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        run: helm package ./
+
+      - name: Login to quay.io
+        if: github.event_name == 'push'
+        env:
+          quay_user: ${{ secrets.QUAY_USER }}
+          quay_password: ${{ secrets.QUAY_PASSWORD }}
+        run: docker login -u="${quay_user}" -p="${quay_password}" quay.io
+
+      - name: Login to quay.io
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply .github/workflows/chart-build.yaml]: Done

**Review by ReToCode — APPROVED** [review]
LGTM

**Pull request merged**